source/ide/ide_methods.bas

```diff
@@ -13355,7 +13351,7 @@ SUB IdeMakeContextualMenu
 
             v = 0
             CurrSF$ = FindCurrentSF$(idecy)
-            IF validname(a2$) THEN v = HashFind(a2$, HASHFLAG_LABEL, ignore, r)
+            IF NOT Error_Happened THEN v = HashFind(a2$, HASHFLAG_LABEL, ignore, r)

```

Comment:
This change should be described

Reply:
I didn't know about this
Never changed it
I think some glitch happened

Reply:
This is likely the result of making changes to an older version of the repo. Make sure to always work on the development branch, and also to always have the latest commit locally.